lib/analysis.js

```diff
@@ -0,0 +1,46 @@
+const alex = require('alex')
+
+function initialize(context) {
+  const processBlob = (async function processBlob(owner, repo, tree_sha, filename, file_sha) {
```

Comment:
Move `processBlob` to top level to avoid redefining the function every time `initialize` is run:

```js
async function processBlob (...) { }

function intialize (...) {}
```